src/lib/shared/api/supabase.ts

```diff
@@ -1,3 +1,4 @@
+import 'isomorphic-fetch';
```

Comment:
question: why is this necessary? It worked without it previously because SvelteKit already contains an isomorphic implementation of `fetch`

Reply:
Well, Jest gave the error `fetch is not defined`. So, I guess, it doesn't work

Reply:
Aha, I see now why it worked previously. Yes, indeed it seems like Jest doesn't have `fetch`. However, I'm not a fan of including this into our actual codebase.

suggestions:
* move `isomorphic-fetch` to `devDependencies`
* remove the import of `isomorphic-fetch` from the application code
* add `'isomorphic-fetch/fetch-npm-node'` to the `setupFiles` in Jest

Just checked it, works equally well